internal/repository/memory_test.go

```diff
@@ -7,8 +7,6 @@ import (
 	"fyne.io/fyne/storage"
 	"fyne.io/fyne/storage/repository"
 
-	_ "fyne.io/fyne/test"
```

Comment:
I was unclear on what this was for, but cargo-culted it from the storage test. What does this do, and when do we need or not need it?

Reply:
It loads the test app driver that allows GUI tests to run headless.
In this instance it created a loop because test needed a repository to be loaded.
I think removing it is sensible and does not break anything.